design/css/style.css

```diff
@@ -0,0 +1,98 @@
+/* General styles */
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  width: 375px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  margin: auto;
+  position: relative;
+  background-color: #e5e5e5;
+}
```

Comment:
Your design should stick as much as possible to the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) 😊

- [ ] Your paragraph color is not the same as in the Figma design 🙃 
- [ ] Your `LET'S CONNECT` button does not have the same color as in the original design 🙃 
- [ ] You don't need the width to your body, that creates some unnecessary spaces 🙃 
- [ ] Also you don't need the height to the top of your website 🙃 

Please kindly fix those points that I mention above 😉 

Your design

<img width="309" alt="Screen Shot 2022-09-21 at 18 30 23" src="https://user-images.githubusercontent.com/74501523/191567016-6dfb7caa-630b-4652-84a6-64997e6c4b20.png">

Figma design

<img width="365" alt="Screen Shot 2022-09-21 at 17 50 26" src="https://user-images.githubusercontent.com/74501523/191567072-d0fe975e-5442-42b8-8ae9-70e15542310b.png">

Reply:
I believe that this is because the items were aligned to the center, so I deleted the alignment so that the card sticks to the top of the page.
This is my last trial for the project is there anything else you think I should do to improve it?